rpm/Makefile

```diff
@@ -3,7 +3,6 @@ source:
 	# this is run from inside our collectd fork so we clone a clean copy to work off of
 	git clone ../../ collectd-$(VERSION).git
 	pushd collectd-$(VERSION).git ; \
-	./clean.sh && ./build.sh && git commit -a -m "Clean and build" ; \
```

Comment:
No, the old logic relied on the HEAD of the git repo to have the clean state, while the new logic re-generates those files during the build, so there's no need to commit them. I think the `git commit` approach was broken anyway, because those files are in `.gitignore`.